### ISSUE TITLE
Different Languages

### ISSUE THREAD

**Issue by nikusha37** [amueller/word_cloud#238]
Hello. Thank you for this great library.

The issue is that I am trying to create it from Georgian words (ქართული სიტყვები). Goergian has its own alphabet. But it is showing me rectangular images instead of words.

Can I do it with this library?

**Comment by amueller** [comment]
you just need to use a font that contains these characters.

**Comment by amueller** [comment]
I think it would be good to use a default font that covers "most languages" as much as possible, see https://github.com/amueller/word_cloud/pull/219#issuecomment-272301932

**Comment by amueller** [comment]
also see discussion in #315 on arabic fonts.

**Comment by amueller** [comment]
@lingdoc yes I've looked into those a while ago. Unfortunately they are too big to include in the package (noto, that is). Also, they are per-language. I thought about combining several of the noto fonts into a single font to ship with wordcloud, but it would be hard to decide what to include, and Chinese is pretty big (and one of the most asked-for languages)

**Comment by lingdoc** [comment]
@amueller understood - I'm definitely not suggesting you include them directly in the package. The links themselves, though, might be useful resources to include in the documentation or on the landing page for users who want to create wordclouds for languages with a non-Roman font.

**Comment by amueller** [comment]
Pr to the docs very welcome! We should have a FAQ
